obspy/clients/fdsn/tests/test_mass_downloader.py

```diff
@@ -22,6 +22,11 @@
 from socket import timeout as socket_timeout
 import tempfile
 import unittest
+import sys
+if sys.version_info.major == 2:
```

Comment:
Empty line after the imports, please. Also please move the new `import sys` to after `from socket ...`